warehouse/transformer/transformer_fuzz_test.go

```diff
@@ -0,0 +1,668 @@
+package transformer
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/araddon/dateparse"
+	"github.com/ory/dockertest/v3"
+	"github.com/stretchr/testify/require"
+	"github.com/tidwall/gjson"
+	"github.com/tidwall/sjson"
+
+	"github.com/rudderlabs/rudder-go-kit/logger"
+	"github.com/rudderlabs/rudder-go-kit/stats"
+	transformertest "github.com/rudderlabs/rudder-go-kit/testhelper/docker/resource/transformer"
+
+	backendconfig "github.com/rudderlabs/rudder-server/backend-config"
+	ptrans "github.com/rudderlabs/rudder-server/processor/transformer"
+	"github.com/rudderlabs/rudder-server/utils/misc"
+	"github.com/rudderlabs/rudder-server/utils/types"
+	"github.com/rudderlabs/rudder-server/warehouse/transformer/testhelper"
+	whutils "github.com/rudderlabs/rudder-server/warehouse/utils"
+)
+
+var (
+	reDateTime = regexp.MustCompile(
```

Comment:
There is already a `reDateTime` defined in utils package. Can that be used instead ?

Reply:
We can use `ValidTimestamp` exported function but here we wanted to capture group values as well from the regex, Since it's just for test, I think it should be fine. 
I don't want to make that regex in the utils package exported.